src/components/Logo.js

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+
+export default function Logo() {
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 1085.7 178.1"
+      alt="Apartment List"
+      className="logo"
+    >
+      <g data-name="Layer 2">
+        <g data-name="Layer 1">
+          <path
+            d="M86.3 127.4H73.8a1.4 1.4 0 0 1-1.3-.9l-8.2-21.1a1.4 1.4 0 0 0-1.3-.9H24.2a1.4 1.4 0 0 0-1.3.9l-8.2 21.1a1.4 1.4 0 0 1-1.3.9h-12a1.4 1.4 0 0 1-1.3-1.9l34.4-87a1.4 1.4 0 0 1 1.3-.9H52a1.4 1.4 0 0 1 1.3.9l34.4 87a1.4 1.4 0 0 1-1.4 1.9zM28.6 90.8h30L43.6 52zM157.8 96.7c0 18.8-13.5 32.3-30.7 32.3-8.7 0-15-3.5-19.2-8.9v31.5a1.4 1.4 0 0 1-1.4 1.4H95.9a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v5.9c4.2-5.4 10.5-8.9 19.2-8.9 17.3 0 30.8 13.5 30.8 32.3zm-50.9 0c0 11.2 7.4 19.7 18.5 19.7s18.6-8.9 18.6-19.7S137 77 125.5 77s-18.6 8.5-18.6 19.7zM225.9 127.4h-10.6a1.4 1.4 0 0 1-1.4-1.4v-6c-4.2 5.4-10.6 9-19.3 9-17.2 0-30.7-13.5-30.7-32.3s13.5-32.3 30.7-32.3c8.7 0 15.1 3.6 19.3 9v-6a1.4 1.4 0 0 1 1.4-1.4h10.6a1.4 1.4 0 0 1 1.4 1.4V126a1.4 1.4 0 0 1-1.4 1.4zm-48.2-30.7c0 10.8 7 19.7 18.5 19.7s18.5-8.5 18.5-19.7S207.3 77 196.2 77s-18.5 8.9-18.5 19.7zM277.2 67.3l-2.5 10.5a1.4 1.4 0 0 1-1.9.9 25.5 25.5 0 0 0-8.7-1.6c-8.5 0-14.6 6-14.6 16.5V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v5.5c4-5.8 10.3-8.5 17.8-8.5a27.9 27.9 0 0 1 9.1 1.3 1.4 1.4 0 0 1 1 1.6zM332.7 78.2h-22.2v26.7c0 8.3 4.4 11.5 9.7 11.5a23.6 23.6 0 0 0 11.2-3.5 1.3 1.3 0 0 1 1.8.6l4.2 8.4a1.3 1.3 0 0 1-.5 1.7A32.8 32.8 0 0 1 319 129c-13.9 0-22.1-7.5-22.1-23.8v-27h-10.8a1.3 1.3 0 0 1-1.3-1.3v-9.6a1.3 1.3 0 0 1 1.3-1.3H297V48.7a1.3 1.3 0 0 1 1.3-1.3h10.8a1.3 1.3 0 0 1 1.3 1.3V66h22.2a1.3 1.3 0 0 1 1.3 1.3v9.6a1.3 1.3 0 0 1-1.2 1.3zM445.7 88.8V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1S402 82.8 402 93.4V126a1.4 1.4 0 0 1-1.4 1.4H390a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1s-15.9 5.7-15.9 16.2V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.5a1.4 1.4 0 0 1 1.4 1.4v6.1c4.9-7.2 12.6-9.1 18.8-9.1 9.4 0 17 4.2 21.2 11 5.3-8.8 14.8-11 22.4-11 14.8 0 25.1 10.1 25.1 24.4zM511.8 96.6c0 1.3-.1 2.5-.2 3.7a1.4 1.4 0 0 1-1.4 1.3h-44c1.8 11 9.6 15.6 18.3 15.6a29.2 29.2 0 0 0 16.7-5.6 1.3 1.3 0 0 1 1.9.3l5.2 7.4a1.4 1.4 0 0 1-.3 1.8c-7.3 5.8-15.4 7.9-24.3 7.9-18.4 0-31.4-12.8-31.4-32.3s12.5-32.3 30.6-32.3 28.9 13.1 28.9 32.2zm-45.6-5.2h31.9c-1.5-9.6-7.1-15.1-15.6-15.1s-14.9 6.2-16.3 15.1zM575.4 88.8V126a1.4 1.4 0 0 1-1.4 1.4h-10.7a1.4 1.4 0 0 1-1.4-1.4V93.2c0-10.3-5.3-16.1-14.3-16.1s-15.9 5.7-15.9 16.2V126a1.4 1.4 0 0 1-1.4 1.4h-10.6a1.4 1.4 0 0 1-1.4-1.4V67.4a1.4 1.4 0 0 1 1.4-1.4h10.6a1.4 1.4 0 0 1 1.4 1.4v6.1c4.9-7.2 12.6-9.1 18.8-9.1 14.6 0 24.9 10.1 24.9 24.4z"
+            fill="#210d36"
```

Comment:
Suggestion: allow the fill values to be passed int through props, with the current values as default

Reply:
Perfect for skinning the site for holidays!

<img width="192" alt="Screen Shot 2020-01-23 at 5 11 56 PM" src="https://user-images.githubusercontent.com/452284/73037357-a0186a00-3e03-11ea-867e-38b76e9885f1.png">